verifiable-credentials-2019.html

```diff
@@ -173,18 +173,13 @@ <h3>
             Other Deliverables
           </h3>
           <dl data-query="active-specifications">
-            <dt><a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Credentials Use Cases</a></dt>
+            <dt><a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Claims Use Cases</a></dt>
```

Comment:
This change (and others) is undoing previous commits.

Reply:
Thanks, was wondering why the diff was showing up blank.